api/prisma/migrations/18_copy_of_reference/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "listings" ADD COLUMN     "copy_of_id" UUID;
+
+-- AddForeignKey
+ALTER TABLE "listings" ADD CONSTRAINT "listings_copy_of_id_fkey" FOREIGN KEY ("copy_of_id") REFERENCES "listings"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
should this `ON UPDATE CASCADE` be part of this? I don't think updates to one of the listing should affect the other?